### ISSUE TITLE
MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 message listeners added to [Worker]

### ISSUE THREAD

**Issue by trivikr** [microsoft/parallel-prettier#7]
I noticed this issue while running parallel-prettier on ~16000 files in https://github.com/aws/aws-sdk-js-v3/pull/1949

<details>
<summary>System Information</summary>

* Operating System: Amazon Linux 2 x86_64
* Host type: c5.4xlarge (16 Virtual CPUs, 32.0 GiB Memory, 160 GiB SSD (EBS) Storage)
* Node.js version: v14.15.4

</details>

Command run was equivalent to:
```console
node --trace-warnings ./node_modules/.bin/pprettier --write **/*.{ts,js,md,json}
```

```console
(node:697) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 message listeners added to [Worker]. Use emitter.setMaxListeners() to increase limit
    at _addListener (events.js:390:17)
    at Worker.addListener (events.js:406:10)
    at setupSubscription (/local/home/trivikr/workspace/aws-sdk-js-v3/node_modules/rxjs/internal/observable/fromEvent.js:38:19)
    at Observable._subscribe (/local/home/trivikr/workspace/aws-sdk-js-v3/node_modules/rxjs/internal/observable/fromEvent.js:25:9)
    at Observable._trySubscribe (/local/home/trivikr/workspace/aws-sdk-js-v3/node_modules/rxjs/internal/Observable.js:44:25)
    at Observable.subscribe (/local/home/trivikr/workspace/aws-sdk-js-v3/node_modules/rxjs/internal/Observable.js:30:22)
    at MapOperator.call (/local/home/trivikr/workspace/aws-sdk-js-v3/node_modules/rxjs/internal/operators/map.js:32:23)
    at Observable.subscribe (/local/home/trivikr/workspace/aws-sdk-js-v3/node_modules/rxjs/internal/Observable.js:25:31)
    at FilterOperator.call (/local/home/trivikr/workspace/aws-sdk-js-v3/node_modules/rxjs/internal/operators/filter.js:29:23)
    at Observable.subscribe (/local/home/trivikr/workspace/aws-sdk-js-v3/node_modules/rxjs/internal/Observable.js:25:31)
```

The error appears around 16 times, equal to number of virtual CPUs.

**Comment by connor4312** [comment]
Thanks for the issue. This is not particularly concerning if files are read faster than they are formatted. But I will take care of it.

**Comment by connor4312** [comment]
Published in 2.0.1
